src/providers/delimitedtext/qgsdelimitedtextsourceselect.cpp

```diff
@@ -860,9 +861,9 @@ void QgsDelimitedTextSourceSelect::updateCrsWidgetVisibility()
   textLabelCrs->setVisible( !geomTypeNone->isChecked() );
 }
 
-QString QgsDelimitedTextSourceSelect::url( bool skipOverriddenTypes )
+QString QgsDelimitedTextSourceSelect::url( QgsDelimitedTextFile *file, bool skipOverriddenTypes )
```

Comment:
Can we take a reference instead of pointer so we have no doubt about memory management of file ?